src/application-status-overlay/ApplicationStatusOverlayProvider.jsx

```diff
@@ -53,6 +68,23 @@ const ApplicationStatusOverlayProvider = ({ children, scrollRefCallback, ...cust
   const registeredTitle = registeredStatusOverlay[lastRegisteredStatusOverlayKey] && registeredStatusOverlay[lastRegisteredStatusOverlayKey].title;
   const registeredVariant = registeredStatusOverlay[lastRegisteredStatusOverlayKey] && registeredStatusOverlay[lastRegisteredStatusOverlayKey].variant;
 
+  useEffect(() => {
+    // eslint-disable-next-line no-prototype-builtins
+    if (!Element.prototype.hasOwnProperty('inert')) {
+      // IE10 throws an error if wicg-inert is imported too early, as wicg-inert tries to set an observer on document.body which may not exist on import
+      // eslint-disable-next-line global-require
+      require('wicg-inert/dist/inert');
+    }
+  });
+
+  useEffect(() => {
+    if (registeredStatusOverlayKeys.length > 0) {
```

Comment:
nit, we can simplify this to a length check
```suggestion
    if (registeredStatusOverlayKeys.length) {
```

Reply:
Changed here : https://github.com/cerner/terra-application/pull/47/commits/347793a8fbb9ff8781d9fce146c1f5a351097b4f